torch/_dynamo/variables/nn_module.py

```diff
@@ -991,7 +1036,12 @@ def call_method(
                 hasattr(method, "__code__")
                 and id(method.__code__) in self._nn_module_method_ids()
             ):
-                unimplemented(f"UnspecializedNNModuleVariable missing {name}")
+                unimplemented_v2(
+                    gb_type="UnspecializedNNModuleVariable missing method",
+                    context=f"call_method: {self} {name} {args} {kwargs}",
+                    explanation=f"Dynamo does not support tracing method {name}",
+                    hints=[],
```

Comment:
Might be difficult to debug ( `*graph_break_hints.DIFFICULT`) because we don't really define "unspecialized nn.Module" for the user very well.

Reply:
Thanks for your review! Have updated in this commit: https://github.com/pytorch/pytorch/pull/151895/commits/f32a060ab6f46a4fa04b366d764d0502abbfb68b